packages/grafana-ui/src/components/ButtonCascader/ButtonCascader.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+// @ts-ignore
+import RCCascader from 'rc-cascader';
+
+export interface ButtonCascaderOption {
+  label: string;
+  value: string;
+
+  children?: ButtonCascaderOption[];
+  disabled?: boolean;
+  // Undocumented tooltip API
+  title?: string;
+}
+
+export interface ButtonCascaderProps {
+  options: ButtonCascaderOption[];
+  buttonText: string;
+  disabled?: boolean;
+  expandIcon?: React.ReactNode;
+  value?: string[];
+
+  loadData?: (selectedOptions: ButtonCascaderOption[]) => void;
+  onChange?: (value: string[], selectedOptions: ButtonCascaderOption[]) => void;
+  onPopupVisibleChange?: (visible: boolean) => void;
+}
+
+export const ButtonCascader: React.FC<ButtonCascaderProps> = props => (
+  <RCCascader {...props}>
+    <button className="gf-form-label gf-form-label--btn" disabled={props.disabled}>
```

Comment:
Any chance of using grafana/ui Button component? We should not use `gf-*` anymore